fe/fe-core/src/main/java/com/starrocks/connector/Connector.java

```diff
@@ -31,4 +33,8 @@ public interface Connector {
      * have been returned from the connector.
      */
     default void shutdown() {}
+
+    default CloudConfiguration getCloudConfiguration() {
```

Comment:
I think the definition in connector isn't appropriate.  Maybe it belongs to metadata